src/main/java/org/folio/circulation/domain/notice/NoticeEventType.java

```diff
@@ -16,6 +16,7 @@
   RECALL_TO_LOANEE("Recall loanee"),
   REQUEST_CANCELLATION("Request cancellation"),
   AVAILABLE("Available"),
+  REQUEST_EXPIRATION("Request Expiration"),
```

Comment:
Will these change as well, following the conversations in the storage pull request?

Reply:
Made the second word lowercase for `Request expiration` and `Hold expiration`.